app/controllers/works_controller.rb

```diff
@@ -39,6 +40,7 @@ def create
       upload_service.update_precurated_file_list(added_files_param, deleted_files_param)
       redirect_to work_url(@work), notice: "Work was successfully created."
     else
+      @work_decorator = WorkDecorator.new(@work, current_user)
```

Comment:
It does not seem that we actually using this variable in the method, do we need it?

Reply:
I am sorry, yes, this should be needed. The tests were breaking for me locally, and I find them also breaking on CircleCI when I attempt to remove this from the `#create` action:
https://app.circleci.com/pipelines/github/pulibrary/pdc_describe/4808/workflows/08304c0b-6d44-4e0b-8f60-c609e6468fd9/jobs/17830